plugins/smart_proxy_remote_execution_ssh/ruby-smart-proxy-remote-execution-ssh.postinst

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+# postinst script for ruby-smart_proxy_remote_execution_ssh
+#
+# see: dh_installdeb(1)
+
+set -e
+
+USERNAME="foreman-proxy"
+GROUPNAME="foreman-proxy"
+FOREMAN_PROXY_HOME="/usr/share/foreman-proxy"
+VARDIR="/var/lib/foreman-proxy"
+
+case "$1" in
+  configure)
+    if [ -d $VARDIR ] ; then
```

Comment:
create the dir if it exists? did you want a `! -d $VARDIR` here?

Reply:
Yes, that's what I meant.